src/node/db/PadManager.js

```diff
@@ -170,6 +171,11 @@ exports.sanitizePadId = async (padId) => {
     padId = padId.replace(from, to);
   }
 
+  // If caseInsensitivePadNames is enabled, we need to make sure the padId is lowercase
+  if (caseInsensitivePadNames) {
+    padId = padId.toLowerCase();
+  }
```

Comment:
style nit (I know that most of the other code doesn't follow this suggested style, but I'm working on changing it bit by bit):

```suggestion
  if (caseInsensitivePadNames) padId = padId.toLowerCase();
```

Reply:
applied in 68fa4f19c45f23c01c8f6f0c54793f78cd1ce930